### DISCUSSION TITLE
Character set issue while import [Feb 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5109] [Discord Threads]
> 🙋‍♂️ **Akashkc**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Acedia**  
#### Can you confirm character set of underlying database, does it allow required characters?  
#  
> 🙋‍♂️ **Akashkc**  
#### Yes, It does  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1061250558964535427)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
